addons/message_upgrade_to_annual_plan/manifest.json

```diff
@@ -49,7 +49,7 @@
         "id": "c_4",
         "type": "button",
         "style": "link",
-        "content": "Get help",
+        "content": "Help",
```

Comment:
This needs a new ID.

Reply:
Fixed - changed everything to having new IDs